src/shared_modules/utils/tests/stringHelper_test.cpp

```diff
@@ -260,3 +260,23 @@ TEST_F(StringUtilsTest, findRegexInStringExtractingSecondGroup)
     EXPECT_TRUE(Utils::findRegexInString(valueToCheck, matchedValue, regex, 2));
     EXPECT_EQ(matchedValue, "extracted");
 }
+
+TEST_F(StringUtilsTest, stringIsNumberFalse1)
+{
+    EXPECT_FALSE(Utils::isNumber("random_string"));
+}
+
+TEST_F(StringUtilsTest, stringIsNumberFalse2)
+{
+    EXPECT_FALSE(Utils::isNumber("r4nd0m_57r1n9"));
+}
+
+TEST_F(StringUtilsTest, stringIsNumberFalse3)
+{
+    EXPECT_FALSE(Utils::isNumber(""));
+}
+
```

Comment:
Instead of creating more than one test that checks for the same thing, why not do the following?

```c++
TEST_F(StringUtilsTest, stringIsNumberFalse)
{
    std::array<std::string, 3> data{"random_string", "r4nd0m_57r1n9", ""};
    for (const auto& it : data)
    {
        EXPECT_FALSE(Utils::isNumber(it));
    }
}
```

Reply:
We should discuss this with the team, I follow the same approach I saw in other tests, keeping every single test separate. Also, this change was approved in another PR.